metrics/test/test_argument_count.py

```diff
@@ -1,21 +1,22 @@
 from pathlib import Path
-import subprocess
 import unittest
 import os
+from byoqm.source_coordinator.source_coordinator import SourceCoordinator
+from metrics.argument_count import ArgumentCount
 
 
 class TestArgumentCount(unittest.TestCase):
     def setUp(self):
         os.chdir("../../")
```

Comment:
Consider adding clarifying comment as to why os.chdir is necessary, because to me it is a little wonky but critical